src/modules/display-popup.js

```diff
@@ -1,41 +1,77 @@
+import { commentsData, addComment } from './comments.js';
+
 const mainContainer = document.querySelector('main');
 const popupContainer = document.createElement('section');
 popupContainer.className = 'popup-cont';
 
-export const display = (img, title, genres, lang, days, time, rating, status) => {
+export const display = (img, title, genres, lang, days, time, rating, sts, id) => {
```

Comment:
- [ Optional] I would suggest that you use more descriptive variable to describe the `sts` so as to make the code more readable and understandable